FWCore/Integration/test/testNoProcessFallback1_cfg.py

```diff
@@ -0,0 +1,26 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("FALLBACK1")
+from FWCore.Modules.modules import EmptySource
+process.source = EmptySource(
+    numberEventsInRun = 8,
+    numberEventsInLuminosityBlock = 8,
+)
+
+process.maxEvents.input = 8
+
+from FWCore.Modules.modules import timestudy_SleepingProducer
+process.intProducer = timestudy_SleepingProducer( ivalue = 1, eventTimes = [0] )
```

Comment:
Is the `timestudy::SleepingProducer` used only to produce `int`, i.e. the sleeping functionality is not really used? (as opposed to `IntProducer` that produces `IntProduct`)